help/en/html/doc/Technical/NetBeans.shtml

```diff
@@ -391,18 +391,20 @@
           Windows using NSIS</a></h2>
 
           <p>It is possible for NetBeans users to create a release
-          package for use on the Windows platform. This requires
-          installation of the Nullsoft Scriptable Install System
-          (NSIS), which can be downloaded from <a href=
-          "http://sourceforge.net/projects/nsis">the NSIS project
-          web site</a> on SourceForge.</p>
-
-          <p>Once NSIS is installed on the computer, it is
-          necessary to create a file called local.properties in the
-          same directory as JMRI's build.xml file. This file is
+          package for use on the Windows platform. JMRI requires use of a special NSIS build, 
```

Comment:
This statement is not correct for building Windows installer packages.

The special build is only needed for compilation of the LaunchJMRI.exe launcher.

Reply:
See comment https://github.com/JMRI/JMRI/issues/6488#issuecomment-456859433

Reply:
I've made some non-trivial changes to reflect the important difference between creating the Windows installer and creating the Windows launcher.  Hopefully this is better!

Reply:
@mattharris please check, and approve if OK